msteams-platform/toolkit/debug-overview.md

```diff
@@ -119,7 +115,7 @@ Here's a list of debug scenarios that you can use:
 
 <summary><b>Skip prerequisite checks</b></summary>
 
-In `.fx/configs/tasks.json` under `"Validate & install prerequisites"` > `"args"` > `"prerequisites"`, update the prerequisite checks you wish to skip.
+In `.vscode/tasks.json` under `"Validate prerequisites"` > `"args"` > `"prerequisites"`, update the prerequisite checks you wish to skip.
```

Comment:
Replace the screenshot below with 
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/37978464/c8452f87-2c84-4656-bc5b-27eb148f0b35)

Two changes:
- update the label
- remove the "ngrok"

Reply:
Updated